ext/CreateParametersExt.jl

```diff
@@ -165,11 +165,22 @@ function FarquharParameters(
     )
     parameters = CP.get_parameter_values(toml_dict, name_map, "Land")
     FT = CP.float_type(toml_dict)
-    MECH = typeof(mechanism)
+    if maximum(is_c3) > 1
```

Comment:
What was the motivation for doing the rounding here, rather than having someone pass in 1 or 0? Since it should represent a bool.

That is, is_c3 should be zero or 1, and not between zero and 1

Reply:
The input map from the NetCDF has 0s and 1s everywhere, but that may not be true after regridding. I thought the constructor should take care of that, but that can also be left to the user.

Reply:
That makes sense to me, thanks for explaining. Maybe we could leave a comment to that effect?